lib/widgets/breez_avatar.dart

```diff
@@ -126,12 +126,20 @@ class _VendorAvatar extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Container(
+    return avatarURL.contains("fastbitcoins") ? CircleAvatar(
```

Comment:
I think it is better to extract out to two components one for each vendor, more explicit and clear this way.